Xamarin.Essentials/Types/Brightness.shared.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+#if WINDOWS_UWP
+using UwpBrightness = Windows.Graphics.Display.BrightnessOverride;
+#endif
+
+namespace Xamarin.Essentials
+{
+    [Preserve(AllMembers = true)]
```

Comment:
I think this attribute is not needed

Reply:
Yep. Not anymore. I think we needed it in the past due to bad things or something.